src/features/pass-survey/ui/ActivityStepper.tsx

```diff
@@ -19,6 +20,7 @@ import {
   StepperPayload,
   XStack,
 } from '@shared/ui';
+import TimeRemaining from '@shared/ui/TimeRemaining.tsx';
```

Comment:
Please correct the import. It should be
```suggestion
import { TimeRemaining } from '@shared/ui';
```